### DISCUSSION TITLE
Can I install extensions in Firefox?

### DISCUSSION THREAD

**Discussion by windchime-yk** [thedevs-network/kutt#670] [Q&A] [answered]
Hi, there.
Currently, when I access the "Add-ons for Firefox" page from the official Kutt website, it says that the page cannot be found.
https://addons.mozilla.org/en-US/firefox/addon/kutt/
What is the reason for this? Is there any plan to reopen it?

Thanks.

**Comment by poeti8** [comment, marked as answer]
Unfortunately Firefox is taking a long time to approve our update for some reasons. I'll try to follow them on this.
